SpatialGDK/Documentation/content/get-started/dependencies.md

```diff
@@ -0,0 +1,27 @@
+# Get started: 1 - Get the Dependencies
+
```

Comment:
Suggest add a tiny bit of text...something friendly like:

"To get going with the GDK, you need the right hardware and software."